docs/_newsfragments/1194-cors.feature.rst

```diff
@@ -0,0 +1,3 @@
+A new kwarg, ``cors_enable``, was added to the :class:`falcon.API` initializer.
+``cors_enable`` can be used to enable a simple blanket CORS policy for all
+routes. (See also: :ref:`cors`.)
```

Comment:
A minor nitpick. Could we possibly generalize _all routes_ to _all responses_?
Since the CORS middleware would also apply to sink output, and pretty much almost everything barring higher hierarchy `process_response` middlewares.

Reply:
Good point.

Reply:
Updated.

Reply:
Good catch wrt other places inside source code :+1: When suggesting the change I was totally focusing on just the towncrier note